vendor/gorepomanifest.json

```diff
@@ -31,8 +31,8 @@
  },
  "github.com/nishanths/fullstory": {
   "git-remote": "https://github.com/nishanths/fullstory",
-  "current-version-git-hash": "5c231e3c1db842ac29ef823e1eb75ecb04808614",
-  "content-hash": "90e212ed048fb0cdc0e362c874147a7365fed7cf"
+  "current-version-git-hash": "272735d360883d9fb41d542b464bc6cdc4f517ee",
```

Comment:
Out of curiosity, is there a reason to vendor anything anymore? Do we need to pin some of these dependencies to *older* versions than their respect `master`?

I think it may be simplest at this point to just remove the `vendor` folder entirely...

Reply:
👍